### DISCUSSION TITLE
Details about pretrained recognition model.

### DISCUSSION THREAD

**Discussion by helpmefindaname** [mindee/doctr#1392] [Q&A] [answered]
Hello,

I want to reproduce the `crnn_vgg16_bn` recognition model with a German vocab. Looking at the [documentation](https://mindee.github.io/doctr/modules/models.html#doctr.models.recognition.crnn_vgg16_bn) which lead to the CRNN paper, describing that it was trained on SynthText and evaluated on other datasets.

I checked out the generator from [SynthText](https://github.com/LaJoKoch/SynthTextGerman) and generated a German dataset and trained a model for a week, however my german `crnn_vgg16_bn` performs significantly worse on our private evaluation data.

So I'd like to have some clarifications:
Is the `crnn_vgg16_bn` model trained purely on SynthText or is there other data added?
If it was trained on SynthText only, how many epochs of the generated 8M images were trained? Or how many training steps were taken?
If it was trained on other data, is that dataset public?

My goal is for it to work on invoices & receipts, but if I manage to train the model well enough to work well on them without creating a private dataset, I could publish it afterwards.

**Comment by felixT2K** [comment, marked as answer]
Hi @helpmefindaname 👋🏼,

All detection and recognition models provided by docTR are trained on a mindee internal real dataset 
det: ~130K samples
rec: ~10M samples

Unfortunately, the dataset contains sensitive data and cannot be shared.

But if we can collect some german open data we could talk about preparing a dataset or maybe an european dataset / data collection
(If you know some sources feel free to share)

The idea behind is if we have enough data i could go ahead and label all these data with Azure Document AI - afterwards shared public - but the first step would be to collect some raw data

Best regards
Felix

**Reply by helpmefindaname** [reply]
Hi @felixT2K
thank you for the clarifications,
so I guess it will be harder to replicate that, as I imagined before.

Sadly I don't have much experience with data collection and cannot help with that.

Best regards
Benedikt
